src/Polly.Core/Hedging/HedgingStrategyOptions.TResult.cs

```diff
@@ -0,0 +1,92 @@
+using System.ComponentModel.DataAnnotations;
+using Polly.Strategy;
+
+namespace Polly.Hedging;
+
+/// <summary>
+/// Hedging strategy options.
+/// </summary>
+/// <typeparam name="TResult">The type of result these hedging options handle.</typeparam>
+public class HedgingStrategyOptions<TResult> : ResilienceStrategyOptions
+{
+    /// <summary>
+    /// Initializes a new instance of the <see cref="HedgingStrategyOptions{TResult}"/> class.
+    /// </summary>
+    public HedgingStrategyOptions() => StrategyType = HedgingConstants.StrategyType;
+
+    /// <summary>
+    /// Gets or sets the minimal time of waiting before spawning a new hedged call.
+    /// </summary>
+    /// <remarks>
+    /// Default is set to 2 seconds.
+    ///
+    /// You can also use <see cref="TimeSpan.Zero"/> to create all hedged tasks (value of <see cref="MaxHedgedAttempts"/>) at once
+    /// or <see cref="HedgingStrategyOptions.InfiniteHedgingDelay"/> to force the hedging strategy to never create new task before the old one is finished.
+    ///
+    /// If you want a greater control over hedging delay customization use <see cref="HedgingDelayGenerator"/>.
+    /// </remarks>
+    public TimeSpan HedgingDelay { get; set; } = HedgingConstants.DefaultHedgingDelay;
+
+    /// <summary>
+    /// Gets or sets the maximum hedged attempts to perform the desired task.
+    /// </summary>
+    /// <remarks>
+    /// Default set to 2.
+    /// The value defines how many concurrent hedged tasks will be triggered by the strategy.
+    /// This includes the primary hedged task that is initially performed, and the further tasks that will
+    /// be fetched from the provider and spawned in parallel.
+    /// The value must be bigger or equal to 2, and lower or equal to 10.
```

Comment:
Why is 10 the hard-limit?

Reply:
This was the value we picked in our internal repo. 

Essentially, hedging is an expensive operation so you want to limit the number of concurrent executions.